crates/sui-graphql-rpc/src/server/builder.rs

```diff
@@ -163,6 +164,19 @@ impl ServerBuilder {
     }
 }
 
+async fn get_schema() -> impl axum::response::IntoResponse {
+    let schema = include_str!("../../schema/current_progress_schema.graphql").to_string();
```

Comment:
What will we use this for, instead of the `__schema` query on the server itself?

Reply:
For context, the idea was to offer the schema in the documentation page in our new docs. But we want to make sure that whatever nightly changes we do and which are not deployed will not affect users' experience. That is, whatever schema is running on the server should be discoverable by the user and we should document it.

Reply:
We should absolutely document the schema in our new docs site (although for me this is a 2.0 roadmap item, not 1.5), but today, GraphiQL has a docs explorer that allows people to interactively explore the schema, as returned by the `__schema` query, within its own UI, so that's where I get confused what the benefit is of also offering a static version of this schema on a GET request -- an up-to-date version of it is already available in a more interactive and easy to access form in the existing deployment.

Reply:
@amnn indeed we can get the schema with `__schema` but its not as pretty as we have it in our codebase.
This endpoint was a convenience feature which allows folks to get a pretty human-readable view for uses like embedding the URL. in docs etc

Reply:
But my point is that the textual schema is also less pretty/useful/accessible than the output generated by GraphiQL from the `__schema` output, which we already have and can point people to.